### PR TITLE
update readme and add folder for screenshots

### PR DESCRIPTION
**Pull request by devthehuman**
## What's this PR do?
* Updated my readme from a one sentence description to (hopefully) a thorough introduction to my app and how to set it up/run it. I also added a folder within the assets folder to hold the screenshots of the app. 

## Pre-Merge TODOs
 - [ ] Screenshots Included
 - [ ] Tests Added
 - [ ] Updated Documentation
 - [x] Add follow up issues?
* The paths to the screenshots may need to be changed from local paths to github urls
 - [ ] 2 PR reviews completed

## Learning
* This is my first attempt at writing a decent readme for a repo, so any comments or suggestions will help me learn a ton.

## Where should the reviewer start?
* Just review the readme!

## Any background context you want to provide?
* n/a

## Screenshots (if appropriate)
* n/a

## What gif best describes this PR or how it makes you feel?
* n/a

### CODE REVIEW

**Comment by devthehuman** [comment]
@tman22

File: tests/components/Controls.test.js [hunk 1]
```diff
@@ -85,6 +85,7 @@ describe('Controls', () => {
     searchButton.simulate('click');
 
     expect(mockSubmit).toHaveBeenCalledTimes(1);
+    expect(mockSubmit).toHaveBeenCalledTimes(1);
```
Comment (JohnBinning) [inline tests/components/Controls.test.js]: I think you could get rid of one of these lines

File: tests/components/Login.test.js [hunk 2]
```diff
@@ -87,7 +91,11 @@ describe('Login', () => {
       preventDefault: jest.fn()
     });
 
-    expect(jest.fn()).toHaveBeenCalledTimes(3);
+    expect(fetchMock).toHaveBeenCalledTimes(1)
+    expect(fetchMock).toHaveBeenCalledWith('devthehuman, password');
+
+    expect(toggleMock).toHaveBeenCalledTimes(1);
+    expect(toggleMock).toHaveBeenCalledWith('loginIsOpen');
```
Comment (JohnBinning) [inline tests/components/Login.test.js]: I like testing the mock function arguments, good work

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+
+## Project Status
+* This project is currently in development
+* Currently, the working functionality is:
+  * Search for concerts based on city
+  * Create an account or login to your own account
+  * Save events to your calendar
+  * Link for directions to event, as well as a link to buy tickets
+
+* Functionality coming soon:
```
Comment (letakeane) [inline README.md]: Nice touch :)

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+  * Ability to communicate with users attending the same shows
+  * Upload a profile picture
+  * Update profile details
+  * More sections to fill out when creating account (i.e. willing to uber/drive, genre prefs, etc)
+
+## Project Screen Shot(s)
+![1]('/app/assets/screenshots/ss1.png')  
+![2]('/app/assets/screenshots/ss2.png')  
+![3]('/app/assets/screenshots/ss3.png')  
+![4]('/app/assets/screenshots/ss4.png')
```
Comment (letakeane) [inline README.md]: This is just my personal preference, but I think the border-radius should be either fully rounded or just square. Or more intentionally weird? But that's just me :D

Comment (Mickyfen17) [inline README.md]: @devthehuman Not sure that you're images are showing properly.

I get the below image when previewing your README.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,55 @@
...
+
+## Project Screen Shot(s)
+![1]('/app/assets/screenshots/ss1.png')  
+![2]('/app/assets/screenshots/ss2.png')  
+![3]('/app/assets/screenshots/ss3.png')  
+![4]('/app/assets/screenshots/ss4.png')  
+
+## Installation and Setup Instructions
+* Clone this repository
+* npm install
```
Comment (letakeane) [inline README.md]: I'd consider using triple backticks to delineate any terminal commands, just for clarity's sake.

**Comment by devthehuman** [comment]
@tman22, @joshuajhun, @rrgayhart for review

**Pull request merged**